nailgun/entities.py

```diff
@@ -2050,6 +2050,41 @@ def update_payload(self, fields=None):
         """Wrap submitted data within an extra dict."""
         return {u'hostgroup': super(HostGroup, self).update_payload(fields)}
 
+    def path(self, which=None):
+        """Extend ``nailgun.entity_mixins.Entity.path``.
+        The format of the returned path depends on the value of ``which``:
+
+        clone
+            /api/hostgroups/:hostgroup_id/clone
+
+        Otherwise, call ``super``.
+
+        """
+        if which in (
```

Comment:
maybe for this one, since there is only one options, we can do a singular check: ```if a == b```